plaso/parsers/sqlite_plugins/android_burners.py

```diff
@@ -0,0 +1,134 @@
+# -*- coding: utf-8 -*-
+"""SQLite parser plugin for Android communication
+  information (burners) database files."""
+
+from plaso.containers import events
+from plaso.parsers import sqlite
+from plaso.parsers.sqlite_plugins import interface
+
+class AndroidCommunicationInformationEventData(events.EventData):
+  """Android Communication Information event data.
+
+  Attributes:
+    phone_number_id (str): Identifier for the associated phone number.
+    voicemail_url (str): URL for accessing the voicemail
+      associated with the burner.
+    user_id (str): ID of the user associated with the burner.
+    name (str): Name of the burner.
+    alias (str): Alias or nickname for the burner.
+    features (str): Features enabled for the burner.
+    total_minutes (int): Total minutes available for calls on the burner.
+    expiration_date (datetime): Expiration date of the burner.
+    date_created (datetime): Date the burner was created.

```

Comment:
for consistency with the rest of the codebase please use creation_time (also applicable to the other date and time values)